### DISCUSSION TITLE
When is v4.9.7 planned ?

### DISCUSSION THREAD

**Discussion by VincentEggerling** [super-linter/super-linter#3408] [General]
Hello !

I recently opened an Issue on the JSCPD linter https://github.com/kucherenko/jscpd/issues/542 that was blocking my work. I'm happy to see that this bug has been quickly fixed and that the dependabot on the Superlinter project has already pulled the latest change :smiley: (https://github.com/github/super-linter/pull/3400).

I'm now wondering: When do you plan to create a new release of the Superlinter on the Github Action Marketplace so that I can start to use it again on my project ?

Thank you !

**Comment by lindluni** [comment]
I just worked with @admiralAwkbar and got this released @VincentEggerling  😃

**Comment by ferrarimarco** [comment]
Nice! There's a small issue with the release workflow tho.  `v4` and `latest` tags points to v4.9.6 until that issue is fixed. Ref #3412

**Reply by lindluni** [reply]
I manually fixed v4 and latest till we get that part of the release workflows fixed. Thanks for calling that out @ferrarimarco

**Reply by ferrarimarco** [reply]
Ok. Just a notice tho: let's wait before merging more PRs because that workflow retag the latest image.

**Reply by ferrarimarco** [reply]
Wait until that issue is fixed, I mean 😃

**Reply by ferrarimarco** [reply]
Tentative fix here: #3413

**Comment by VincentEggerling** [comment]
Thanks a lot for your quick responses and release :+1:

**Comment by lindluni** [comment]
4.10.0 is released with nearly all dependencies updated.
